main/nagios-plugins/nagios-plugins.post-upgrade

```diff
@@ -0,0 +1,8 @@
+#!/bin/sh
+
+cat <<EOF
+NOTE: Since 2.0.3-r2 all plugins are separated into subpackages
```

Comment:
This shouldn't be printed unconditionally, but I don't see a way to compare old and new versions. I'll merge it as is.